client/rpc/src/system/mod.rs

```diff
@@ -97,6 +103,34 @@ impl<B: traits::Block> SystemApi<B::Hash, <B::Header as HeaderT>::Number> for Sy
 		Receiver(Compat::new(rx))
 	}
 
+	fn system_add_reserved_peer(&self, peer: String)
```

Comment:
The implementation of all methods of this trait is really tedious. It should be auto-generated, but I guess that's a separate task.

Reply:
I 100% agree. The architecture of having the RPC worker forward requests to something that answers them asynchronously is good, though.

Reply:
Yes, I guess we will move to purer architecture with `jsonrpsee`, so I guess no action needed here, I'm just whining :)

Reply:
When I was writing this PR, using the compat between stable futures and 0.1 was a real headache. i took a quick stab at porting over `jsonrpc` to stable futures but then realized it was a decent amount of work.